### PR TITLE
Feature/pop

### PR DESCRIPTION
**Pull request by AndrZakhara**
add method pop

### CODE REVIEW

**Review by PodstDO — APPROVED** [review]

**Review by PodstDO — CHANGES_REQUESTED** [review]
Why `items`?????????????????????

File: src/index.js [hunk 1]
```diff
@@ -24,6 +24,10 @@ class Stack {
     return stack;
   }
 
+  pop() {
+    return this.elements.splice(this.items.length - 1, 1);
```
Comment (Alexandra1003) [inline src/index.js]: change this.items to this.elements

**Review by Alexandra1003 — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (AndrZakhara) [reply src/index.js]: fixed

**Review by PodstDO — APPROVED** [review]

**Review by Alexandra1003 — APPROVED** [review]

**Comment by AndrZakhara** [comment]
add method

**Pull request merged**